src/components/Tag/Tag.tsx

```diff
@@ -17,24 +19,46 @@ export interface NonMutuallyExclusiveProps {
   onClick?(): void;
   /** Callback when remove button is clicked or keypressed. */
   onRemove?(): void;
+  /** Url to  */
+  url?: string;
 }
 
 export type TagProps = NonMutuallyExclusiveProps &
   (
-    | {onClick?(): void; onRemove?: undefined}
-    | {onClick?: undefined; onRemove?(): void}
+    | {onClick?(): void; onRemove?: undefined; url?: undefined}
+    | {onClick?: undefined; onRemove?(): void; url?: string}
   );
 
-export function Tag({children, disabled = false, onClick, onRemove}: TagProps) {
-  const i18n = useI18n();
-
+export function Tag({
+  children,
+  disabled = false,
+  onClick,
+  onRemove,
+  url,
+}: TagProps) {
   const className = classNames(
     styles.Tag,
     disabled && styles.disabled,
-    onClick && styles.clickable,
+    (onClick || url) && styles.clickable,
     onRemove && styles.removable,
+    onRemove && url && styles.segmented,
   );
 
+  if (onClick) {
```

Comment:
This is all basically some refactoring that should keep the old options the same and only change something when `url` is set.

Since `onClick` is the most restrictive prop and using a ternary for this case was complex, I decided to return early here, but this could be moved around in any way in case there are some specific conventions I'm not aware of.